parameter/MDTParamenter_WCTE.txt

```diff
@@ -5,28 +5,45 @@
 < NDigitsStepSize = 5 >
 < NDigitsPostTriggerWindow = +950. >
 < NDigitsPreTriggerWindow = -400. >
-< FailurePostTriggerWindow = +100000. >
+< FailurePostTriggerWindow = +950. > # seems to be +950 by default //+100000.
 < FailurePreTriggerWindow = -400. >
 < FailureTime = 100. >
+< NDigitsWindow = 200. >
 
 # Dark noise setting
-< NumOfTubes = 10184 >
+< NumOfTubes = 2014 >
 < MinTubeID = 1 >
-< MaxTubeID = 10184 >
+< MaxTubeID = 2014 >
 < DarkAddMode = 1 > 
 < DarkRate = 0.0 > // kHz
 < DarkM0WindowLow = 0. > // ns
 < DarkM0WindowUp = 100000. > // ns
 < DarkM1Window = 4000. > // ns
+< NumOfTubes_PMT3inchR12199_02 = 2014 >
+< MinTubeID_PMT3inchR12199_02 = 1 >
+< MaxTubeID_PMT3inchR12199_02 = 2014 >
+< DarkRate_PMT3inchR12199_02 = 0.0 >
 
 # 3" PMT setting
 < TimingResConstant = 1.890 > # TTS(FWHM) = 1ns at 1 charge
 < TimingResMinimum = 0.58 >
 < ScalFactorTTS = 1 > 
+< TimingResConstant_PMT3inchR12199_02 = 1.890 > # TTS(FWHM) = 1ns at 1 charge
+< TimingResMinimum_PMT3inchR12199_02 = 0.58 >
+< ScalFactorTTS_PMT3inchR12199_02 = 1 >
 
 # Digitizer setting
 < DigiHitIntegrationWindow = 200 >  // ns
-< PrecisionTiming = 0.1 > // ns
+< PrecisionTiming = 0.1 > // ns 
 < PrecisionCharge = 0. >
+< ApplyDAQEfficiency = 1 >
 
-< SPECDFFile_PMT3inchR12199_02 = /home/hep/rakutsu/t2k/wcte/mdt_4_wcte/MDT/parameter/SPE_CDF_PMT3inchR12199_02.txt >
+# After pulse setting
+< APDoAdd = 0 >
+
+# Output setting
+< TimeOffset = 950. >
+< FlagMultDigits = 1 >
+
+
+< SPECDFFile_PMT3inchR12199_02 = ./parameter/SPE_CDF_PMT3inchR12199_02.txt >
```

Comment:
does this eventually use `$MDTROOT` in the code (so it can be run anywhere)?

Reply:
add a small function inside `cpp/Configuration` to read `$MDTROOT` whenever possible